CONTRIBUTING.md

```diff
@@ -31,23 +33,30 @@ We recommend you to use our prebuilt Docker image to start your development work
 
 * From VSCode's command menu, run `Reopen in Container` to open your workspace in one of our pre-built Docker containers. Select the correct container config based on your local accelerator (default to `tpu-contributor` if you are not sure).
 
-* Since you are running as root in this container, change ownership of all files:
+* Since you are running as root in this container, teach `git` to recognize the repositories you
+  just cloned (outside of docker) as safe:
 
   ```bash
-  chown -R root:root .
+  git config --global --add safe.directory /workspaces/torch/pytorch
+  git config --global --add safe.directory /workspaces/torch/pytorch/xla
+  git config --global --add safe.directory /workspaces/torch/vision
   ```
 
-* Build PyTorch and PyTorch/XLA:
+* Build PyTorch, TorchVision, and PyTorch/XLA:
 
   ```bash
-  cd pytorch/
+  pushd pytorch
```

Comment:
why use `pushd` instead of `cd`?

Reply:
Usually I prefer `pushd` when navigating deep folders but I think this case there's no obvious advantage, so I'll replace with the more familiar `cd`.